issues.js

```diff
@@ -1,6 +1,12 @@
 const config = require('./config'),
   octokit = require('@octokit/rest')({ headers: config.headers });
 
+
+const removeExcludedLabels = (issues) => {
+  return issues.filter(issue => {
+    return !issue.labels.some(label => config.excludeLabels.includes(label.name.toLocaleLowerCase()));
```

Comment:
We should also be doing `toLocaleLowerCase()` on the `config.excludeLabels` as well just to be sure our logic is robust and handling mixed-case situations